hazelcast/src/main/java/com/hazelcast/spi/impl/transceiver/impl/PacketTransceiverImpl.java

```diff
@@ -66,10 +65,6 @@ public boolean transmit(Packet packet, Connection connection) {
         if (connection == null || !connection.isAlive()) {
             return false;
         }
```

Comment:
I like the following section!!

You can even simplify it to

```
If(connection = null) return false;
return connection.write(packet)
```

Since the write will do an alive check anyway.